### PR TITLE
Update es_US.lang

### PR DESCRIPTION
**Pull request by ShadowOne333**
Updated es_US.lang with the corrections made in #153 and #154, and made a separate file for Latin American spanish.

### CODE REVIEW

**Comment by Cruel** [comment]
Format looks good, will just wait to get a little more input from others.

**Comment by ShadowOne333** [comment]
Sure!
Thanks :)

File: res/romfs/lang/es_US.lang [hunk 1]
```diff
@@ -33,16 +33,16 @@ Descargando las llaves de títulos...
 # src/States/BrowseState.cpp
 ############################
 No title keys found.\nMake sure you have keys in\n%s\n\nManually copy keys to the directory\nor check settings to enter a URL\nfor downloading title keys.
-No se encontraron llaves de títulos.\nVerifique que tenga las llaves en\n%s\n\nCopie manualmente las llaves al\no directorio o verifique las configuraciones\npara ingresar un URL para\no descargar las llaves.
+No se encontraron llaves de títulos.\nVerifique que tenga las llaves en\n%s\n\nCopie manualmente las llaves al\no directorio o verifique las configuraciones\ne ingrese un URL para\ndescargar las llaves.
```
Comment (gemarcano) [inline res/romfs/lang/es_US.lang]: "ne ingrese" debería ser "ni ingrese" -- la palabra "ni" no se le cambia la vocal aunque la próxima palabra comience con una "i".

In English:
"ne ingrese" should be "ni ingrese" -- the word "ni" does not change its vowel even when the following word begins with the letter "i".

Comment (ShadowOne333) [inline res/romfs/lang/es_US.lang]: Estas confundiendo un poco el formato.
La "n" es parte de "\n", el cual es el codigo utilizado para un salto de linea en lenguaje de programacion.
En si, la frase se lee asi:
"...las configuraciones
e ingrese un URL..."

@Cruel He seems to have confused the jump line command with a word, he read it as "ne" but it is actually only an "e".

File: res/romfs/lang/es_US.lang [hunk 2]
```diff
@@ -33,16 +33,16 @@ Descargando las llaves de títulos...
...
 
 No cache entries found\nfor your title keys.\n\nTry refreshing cache in settings.\nIf that doesn't work, then your\ntitles simply won't work with\nfreeShop currently.
-No se encontraron entradas de caché\npara tus llaves de títulos.\n\nIntente atualizar el caché en configuraciones.\nSi eso no funciona, entonces\ntus títulos simplemente no funcionarán\ncon freeShop actualmente.
+No se encontraron entradas de caché\npara tus llaves de títulos.\n\nIntente actualizar el caché en configuraciones.\nSi eso no funciona, entonces\ntus títulos simplemente no funcionarán\ncon freeShop actualmente.
```
Comment (gemarcano) [inline res/romfs/lang/es_US.lang]: s/tus/sus/

File: res/romfs/lang/es_US.lang [hunk 3]
```diff
@@ -53,7 +53,7 @@ Ya instalado: %s
  Cancelar
 
 Are you sure you want\nto continue?
-¿Está seguro de que quieres continuar?
+¿Está seguro de que quiere continuar?
```
Comment (gemarcano) [inline res/romfs/lang/es_US.lang]: s/de//

File: res/romfs/lang/es_US.lang (hunk 1 above)
Comment (gemarcano) [inline res/romfs/lang/es_US.lang]: Yep, that's what happened.

**Comment by Cruel** [comment]
"en fila de instalación" is too long, just make it "en fila".

**Comment by ShadowOne333** [comment]
Huh I tried to change all of the "file de instalacion" instances to just "fila" and I got an error.

**Comment by Cruel** [comment]
I changed something in an external lib that broke it, this didn't do it.

**Pull request merged**